packages/gui/src/components/nfts/NFTDetails.tsx

```diff
@@ -231,6 +231,46 @@ export default function NFTDetails(props: NFTDetailsProps) {
       });
     }
 
+    if (metadata?.preview_image_uris) {
+      const value = metadata?.preview_image_uris.map(
+        (uri: string, idx: number) => {
+          return (
+            <span>
+              &nbsp;
+              <a href={uri} target="_blank">
+                {uri}
+              </a>
```

Comment:
For the links here and below, we can use the `<Link>` component

Reply:
Done.